great_expectations/checkpoint/actions.py

```diff
@@ -48,25 +48,19 @@
 
 @public_api
 class ValidationAction:
-    """Base class for all Actions that act on Validation Results and are aware of a Data Context namespace structure.
+    """
+    ValidationActions define a set of steps to be run after a validation result is produced.
 
-    Args:
-        data_context: Data Context that is used by the Action.
+    Through a Checkpoint, one can orchestrate the validation of data and configure notifications, data documentation updates,
+    and other actions to take place after the validation result is produced.
     """
 
-    def __init__(self, data_context: AbstractDataContext) -> None:
-        """Create a ValidationAction"""
-        self.data_context = data_context
+    def __init__(self) -> None:
+        from great_expectations import project_manager
 
-    @property
-    def _using_cloud_context(self) -> bool:
-        # Chetan - 20221216 - This is a temporary property to encapsulate any Cloud leakage
-        # Upon refactoring this class to decouple Cloud-specific branches, this should be removed
-        from great_expectations.data_context.data_context.cloud_data_context import (
-            CloudDataContext,
-        )
-
-        return isinstance(self.data_context, CloudDataContext)
+        self._using_cloud_context = project_manager.is_using_cloud()
+        self._build_data_docs = project_manager.build_data_docs
+        self._get_docs_sites_urls = project_manager.get_docs_sites_urls
```

Comment:
Moved these all to the base class but perhaps it is better to move them to a `DataDocsAction` base class?

Reply:
That does sound better! Yes, please move them there instead.